MobileSDK_To_AmplifySwift_Guidance.md

```diff
@@ -0,0 +1,161 @@
+
+---
+
+### Analytics
+
+> **Pinpoint deprecation notice** – Pinpoint will be retired Oct 30 2026. AWS End User Messaging is the recommended successor. Plan migrations accordingly.
+
+| AWS SDK for iOS                                                                                                   | Amplify Swift                                                                                                                        |
+|-------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------|
+| [Manually Track Session](https://docs.amplify.aws/gen1/swift/sdk/analytics/getting-started/#add-analytics)         | [Automatically Track Session](https://docs.amplify.aws/gen1/swift/build-a-backend/more-features/analytics/set-up-analytics/#initialize-amplify-analytics) |
+| [Add Analytics](https://docs.amplify.aws/gen1/swift/sdk/analytics/getting-started/#add-analytics)                  | [Add Analytics](https://docs.amplify.aws/gen1/swift/build-a-backend/more-features/analytics/set-up-analytics/#initialize-amplify-analytics)                |
+| [Authentication Events](https://docs.amplify.aws/gen1/swift/sdk/analytics/events/#authentication-events)                  | [Authentication Events](https://docs.amplify.aws/gen1/swift/build-a-backend/more-features/analytics/record-events/#authentication-events)                  |
+| [Custom Events](https://docs.amplify.aws/gen1/swift/sdk/analytics/events/#custom-events)                           | [Custom Events](https://docs.amplify.aws/gen1/swift/build-a-backend/more-features/analytics/record-events/)                                               |
+
+---
+
+### AWS IoT
+
+Amplify currently does **not** expose an IoT category. Use the standalone [**AWS IoT Device SDK for Swift**](https://github.com/aws/aws-iot-device-sdk-swift/blob/main/Package.swift), which works side‑by‑side with Amplify.
+
+```swift
+// Example: Connect & subscribe with AWS IoT Device SDK Swift
+import AwsIot
+...
+```
+
+---
+
+## When Amplify doesn't cover a service
+
+For AWS services without an Amplify category (e.g., Amazon SQS, EventBridge, DynamoDB Streams) import the **AWS SDK for Swift** directly.\
+See runnable examples in the [aws‑doc‑sdk‑examples/swift](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/swift) repository.
+
+> **Tip** – You can share credentials between Amplify and AWS SDK by passing `Amplify.Auth.fetchAuthSession()` credentials into service client configuration.
+
+---
+
+## Migration checklist
+
+1. **Remove** SDK pods / SPM packages for `AWSMobileClient`, `AWSS3TransferUtility`, etc.
+2. **Create or import** your backend in **Amplify Gen 2** console and download the generated configuration (or run `amplify pull`).
```

Comment:
I don't think this is right. Previous customers likely have a gen1 Amplify project. Now if a customer does not have an Amplify project, we should inform to use Gen2